spec/requests/api/v1/business_intelligence/merchant_intel_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+describe 'Single Merchant Business Intelligence' do
+  before(:each) do
+    @invoice = create(:invoice)
+    invoice_items = create_list(:invoice_item, 3, invoice_id: @invoice.id)
+    transactions = create_list(:transaction, 3, invoice_id: @invoice.id)
+    @merchant = Invoice.first.merchant
+  end
+
+  it 'returns total revenue accross all transactions' do
+    get "/api/v1/merchants/#{@merchant.id}/revenue"
+
+    found = JSON.parse(response.body)
+
+    expect(response).to be_success
+    expect(found).to have_key("revenue")
+    expect(found["revenue"]).to be_a(String)
+  end
+
```

Comment:
It would be great if we could also test that the math is working. (We know it is if it passes the spec harness, but we could also test it here)